pkg/extensions/search/utils/db.go

```diff
@@ -0,0 +1,244 @@
+package cveinfo
```

Comment:
this is a search/cve extension so call it cve.go - that it uses a db is a separate thing

Reply:
Ok, I will change that